ncm-systemd/src/main/pan/components/systemd/schema.pan

```diff
@@ -145,6 +145,10 @@ http://www.freedesktop.org/software/systemd/man/systemd.exec.html
 valid for [Service], [Socket], [Mount], or [Swap] sections
 }
 type ${project.artifactId}_unitfile_config_systemd_exec = {
+    'CacheDirectoryMode' ? string # octal notation
```

Comment:
either use a int and print it in octal, or make a new string type (if none exists already) with a regexp that locks this down